src/main/resources/application.yaml

```diff
@@ -38,6 +38,13 @@ spring:
     password: ${POSTGRES_PASSWORD:dbuserprofile}
     properties:
       charSet: UTF-8
+    hikari:
+      minimumIdle: 2
+      maximumPoolSize: 10
+      idleTimeout: 600000
+      poolName: LRDHikariCP
```

Comment:
Pool name is LRDHikari. Shouldn't it be UPHikariCP?

Reply:
yes renamed